src/command/backup.rs

```diff
@@ -61,8 +61,14 @@ pub fn backup(
                 None => continue,
             };
 
+            let screenshot_directory = match config.get_screenshots_directory_for_flavor(&flavor) {
+                Some(path) => path,
+                None => continue,
```

Comment:
Does the screenshot folder always exist? If not, we shouldn't `continue` here unless `ALL` or `screenshot` was passed on the command line. Otherwise it'll cause backup to fail even when the user doesn't want to backup the screenshots folder, if they don't have screenshots.

Reply:
Installed PTR to make sure, it does not always exist. The current implementation will create a screenshots directory however. It wasnt a problem when I tried to backup non-existing directory.

Maybe not a decent solution, what could be a better implementation? Maybe initialize in said task. Example below (sorry about poor indentation lol)

```rust
match backup_folder {
      ....
      BackupFolder::Screenshots => {
        if let Some(screenshot_directory) = config.get_screenshots_directory_for_flavor(&flavor) {
          let screenshots_folder = backup::BackupFolder::new(&screenshot_directory, &wow_directory); 
          if screenshot_directory.exists() {
           src_folders.push(screenshots_folder);
          }
        };
    }
}
```

then do something similar for the "ALL" task

Reply:
Ahh yes, my bad. The function returns None only if the wow directory isn't set for that flavor, not if that folder doesn't exist. 

And you already have a check on `if_exists` before pushing the backup folder. So I think this is actually fine!

Reply:
Cheers! then I'm resolving it :)